Modules/Sources/BuildSettingsKit/BuildSettings.swift

```diff
@@ -29,7 +29,8 @@ public struct BuildSettings: Sendable {
         case .preview:
             return .preview
         case .test:
-            fatalError("BuildSettings are unavailable when running unit tests. Make sure to inject the values manually in system under test.")
+            // TODO: update tests to ensure none of the rely on `BuildSettings` availability as it's incompatible with parallelized tests
+            return .live
```

Comment:
I have to backtrack on the idea of making `BuildSettings.current` unavailable in unit tests. There are too many tests to update and we don't have the time now. We'll just continue to assume that unit tests run in the "WordPress App" environment for now.